app/helpers.php

```diff
@@ -136,3 +136,56 @@ function display_sidebar()
     isset($display) || $display = apply_filters('sage/display_sidebar', false);
     return $display;
 }
+
+/**
+ * Block variables
+ * @return object
+ */
+function block_vars()
+{
+    $editorConfig = json_decode(file_get_contents(__DIR__ .'/../resources/assets/editor.json'));
+
+    $values = (object) [
+        'colors' => gather_colors($editorConfig),
+        'font_sizes' => gather_font_sizes($editorConfig),
+    ];
+
+    return $values;
+}
+
+/**
+ * Gather Colors
+ * @param  object config
+ * @return array colors
+ */
+function gather_colors($editorConfig)
+{
+    $colors = [];
+    foreach ($editorConfig->styles->colors as $name => $color) {
```

Comment:
If the 'editor.json' is missing (or corrupt) this throws an error. Should we check for this?

Reply:
Hey @dsturm these are all great suggestions ✨, but this PR is pretty much not going anywhere and has been left up purely as a resource for others. 

If you submit a PR to my branch I'd accept it but my last commit to this was in May of 2019.

I'm sure a lot of the CSS is super out-of-spec by now. The editor markup has changed a lot. Even the anticipated markup for full/wide align blocks, which you commented on, doesn't match up to how those blocks' actual markup anymore, I don't believe.